extensions/skins/conversation_v1/player.html

```diff
@@ -258,11 +276,16 @@ <h4 class="conversation-skin-responses-dropdown-container" ng-click="toggleShowP
       <span class=" glyphicon glyphicon-arrow-left">
       </span>
     </li>
-    <li ng-repeat="dot in dots track by $index" class="oppia-animate-progress-dot">
+    <li ng-repeat="dot in dots track by $index"
+        class="oppia-animate-progress-dot"
+        ng-show="($index >= leftmostVisibleDotIndex && $index < rightmostVisibleDotIndex)" >
```

Comment:
No need brackets. Also I suggest that rightmostVisibleDotIndex be an inclusive bound (since, given its name, you'd expect it to be one of the dots showing on the page).